### ISSUE TITLE
Question: Troule with repacked installation of DBeaver

### ISSUE THREAD

**Issue by pdanie** [dbeaver/dbeaver#3750]
DBeaver - **Version 5.1.1 CE**
Operating System: **Windows 7 / Windows 8.1 / Windows 10**

We have repacked the _DBeaver_ to be able to ship it with our configuration. But when running the **dbeaver.exe** on some **Windows 7** Machines _DBeaver_ does not start up.

The way we repack _DBeaver_ are:

**o** We run your original installation on a **Windows 10** machine.
**o** We configure the _DBeaver_ to use a configuration directory in the installation directory.
**o** We configure the _DBeaver_ to use a driver directory in the intallation directory.
**o** Our installation copies all the files (program, configuration and drivers).
**o** We finall add a shortcut on that point to the "dbeaver.exe" file.

On most machine, this workes super great, but on some **Windows 7** Machines nothing happens when I try to start the program.

Question - Are there:

**o** Some more considurations that the original _DBeaver_ insallation does on what files to use?
**o** Some files that are installed outside the choosen _DBeaver_ installation directory?
**o** Some registrations of shared installed files needed?
**o** Are there some windows registry values that are essetial that must exist?

I have run the original installer on a _SandboxIE_ environment and all files seems to be installed in the program directory only (and of course configuration and drivers in the user AppData directy). And only windows registry value I have noticed are two values that refer to the path of the **dbeaver.exe** file.

If I on a "trouble" machine run our original _DBeaver_ installer, that installation **workes ok**.

Tips of what I might suffer from or way I can furter test what is amiss would be very appreciated.

**Comment by serge-rider** [comment]
> Some more considurations that the original DBeaver insallation does on what files to use?

No, there are no any specific dependencies. Except Java. But it is already bundled in installer.
BTW what arch do you use (32 or 64 bit)? Are all of your machines have the same arch?

> Some files that are installed outside the choosen DBeaver installation directory?

No.

> Some registrations of shared installed files needed?

No.

> Are there some windows registry values that are essetial that must exist?

No.

DBeaver installer copies files only in target installation directory. DBeaver itself then creates configuration/drivers folders in the user home (or wherever it was configured to).

**Comment by pdanie** [comment]
We have only bundled the 32-bit version of _DBeaver_.

The machine where we do installation are both 32-bit machines and 64-bit machines, with Windows-7, 8, 8.1 and 10.

PS. We have considered to run the _DBeaver_ installation silent within our own installation, but there is still a bug in the _DBeaver_ installation that we can't specify the installation directory (the command-line parameter is ignored).
